components/inputs/test/input-time.test.js

```diff
@@ -102,14 +106,30 @@ describe('d2l-input-time', () => {
 
 		it('should not save input seconds after time changes', async() => {
 			//Seconds are saved when value is assigned directly, not when input by user (for EOD)
-			const elem = await fixture('<d2l-input-time value="11:22:33"></d2l-input-time>');
+			const elem = await fixture(fixtureWithValue);
 			expect(elem.getTime().second).to.equal(33);
 			getInput(elem).value = '11:45 AM';
 			dispatchEvent(elem, 'change', false);
 			await oneEvent(elem, 'change');
 			expect(elem.value).to.equal('11:45:00');
 		});
 
+		it('should update value when dropdown changes', async() => {
+			const elem = await fixture(fixtureWithValue);
+			expect(elem.getTime(elem).second).to.equal(33);
+			getFirstOption(elem).click();
+			await aTimeout(1);
```

Comment:
Can you await the change event instead of using a timeout? Timeouts are generally kind of flaky.